### DISCUSSION TITLE
middleware/mysql/intro/features

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2785] [Announcements]
# middleware/mysql/intro/features

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/mysql/intro/features.html

**Comment by fangbin9527** [comment]
"使用 S erviceMonitor 对接 Prometheus 抓取指标"  中的“S erviceMonitor”多了空格

**Reply by windsonsea** [reply]
Fixed in PR #5296
